src/librustdoc/config.rs

```diff
@@ -797,7 +817,11 @@ impl Options {
 
     /// Returns `true` if the file given as `self.input` is a Markdown file.
     pub(crate) fn markdown_input(&self) -> bool {
```

Comment:
If this returned `Option<&Path>` instead of `bool` **(\*)**

Reply:
Indeed returning `Option<&Path>` is a valid option; Done.